Libraries/Components/Navigation/NavigatorIOS.ios.js

```diff
@@ -270,9 +274,15 @@ var NavigatorIOS = React.createClass({
     barTintColor: PropTypes.string,
 
     /**
-     * The text color of the navigation bar title
+     * The style of the navigation bar title
      */
-    titleTextColor: PropTypes.string,
+    titleStyle: StyleSheetPropType({
+      color: TextStylePropTypes.color,
+      fontFamily: TextStylePropTypes.fontFamily,
+      fontSize: TextStylePropTypes.fontSize,
+      fontStyle: TextStylePropTypes.fontStyle,
+      fontWeight: TextStylePropTypes.fontWeight
```

Comment:
style nit: trailing comma on the last item when each `key: value` pair is on its own line